lib/ansible/plugins/cliconf/vyos.py

```diff
@@ -59,7 +59,7 @@ def edit_config(self, command):
             self.send_command(cmd)
 
     def get(self, command, prompt=None, answer=None, sendonly=False):
-        return self.send_command(to_bytes(command), prompt=to_bytes(prompt), answer=to_bytes(answer), sendonly=sendonly)
+        return self.send_command(command, prompt=prompt, answer=answer, sendonly=sendonly)
```

Comment:
A followup now: Why does `get()` exist? Is it just a naming thing? Should `CliconfBase.send_command` be `_send_command` instead so it isn't exposed to the module?

Reply:
send_command